pipeline-templates/job/deploy.yml

```diff
@@ -0,0 +1,39 @@
+parameters:
+  AksClusterName:
+  AksResourceGroupName:
+  AzureServiceConnection:
+  Environment:
+  KubectlVersion:
+  DependsOn: []

```

Comment:
This parameter isn't being used so can be removed

Reply:
Removed